### ISSUE TITLE
Upgrade to Terraform 0.13

### ISSUE THREAD

**Issue by srueg** [appuio/terraform-openshift4-cloudscale#10]
Upgrade this module to Terraform 0.13: https://www.terraform.io/upgrade-guides/0-13.html

**Comment by srueg** [comment]
Closed by #12

**Comment by nunojusto** [comment]
One thing that happened to me when using the new version.
Because I was using a container terraform run, there was a local docker volume with some wrong values.
And i was receiving this error:
```
Error: Failed to install providers

Could not find required providers, but found possible alternatives:

  hashicorp/cloudscale -> terraform-providers/cloudscale

If these suggestions look correct, upgrade your configuration with the
following command:

The following remote modules must also be upgraded for Terraform 0.13
compatibility:
- module.cluster at
github.com/appuio/terraform-openshift4-cloudscale?ref=master
```
After i removed the local volume, either with `docker volume rm <VOLUME>`or with `docker volume prune` the problem was fixed and all works fine again.
